sdcm/stress_thread.py

```diff
@@ -162,26 +149,47 @@ def _get_available_suboptions(self, node, option, _cache={}):  # pylint: disable
     @staticmethod
     def _disable_logging_for_cs(node, _cache={}):  # pylint: disable=dangerous-default-value
         if not (node.is_kubernetes() or node.name in _cache):
-            node.remoter.run("cp /etc/scylla/cassandra/logback-tools.xml .", ignore_status=True)
+            node.run("cp /etc/scylla/cassandra/logback-tools.xml .", ignore_status=True)
             _cache[node.name] = 'done'
 
-    def _run_stress(self, node, loader_idx, cpu_idx, keyspace_idx):  # pylint: disable=too-many-locals
-        stress_cmd = self.create_stress_cmd(node, keyspace_idx)
+    def _run_stress(self, loader, loader_idx, cpu_idx):
+        pass
+
+    def _run_cs_stress(self, loader, loader_idx, cpu_idx, keyspace_idx):  # pylint: disable=too-many-locals
+        cpu_options = ""
+        if self.stress_num > 1:
+            cpu_options = f'--cpuset-cpus="{cpu_idx}"'
+
+        if loader.is_kubernetes():
+            docker = loader.remoter
```

Comment:
It was called node across the board, and that was confusing.

Since I was under the assumption everything would work based on docker image, I assume it's good enough name.

BTW it's the same name I've used in the other stress command

Reply:
`s-b` doesn't use `docker` naming anymore: https://github.com/scylladb/scylla-cluster-tests/commit/a35bbc32

Reply:
We'll need to align also the rest of the stress code

Reply:
we'll do that slowly, for each stress command as needed (since now no k8s use-case is using any of them)